examples/crm/src/sales/useIsAdmin.tsx

```diff
@@ -0,0 +1,12 @@
+import { useGetIdentity, useGetOne } from 'react-admin';
+import { Sale } from '../types';
+
+export function useIsAdmin() {
+    const { identity } = useGetIdentity();
+    const user = useGetOne<Sale>('sales', { id: identity?.id });
```

Comment:
```suggestion
    const{ data, isPending, error } = useGetOne<Sale>('sales', { id: identity?.id }, { enabled: !!identity);
```

Reply:
This was removed by using permissions instead